lib/SearchAndSort/SearchAndSort.js

```diff
@@ -725,10 +743,11 @@ class SearchAndSort extends React.Component {
               <Button
                 id={`clickable-new${objectName}`}
                 aria-label={ariaLabel}
-                href={this.craftLayerUrl('create')}
                 buttonStyle="primary"
                 marginBottom0
-                onClick={this.addNewRecord}
+                href={createRecordPath ? undefined : this.craftLayerUrl('create')}
```

Comment:
Why do we have `href` as well as `to`?

If we're going to have it at all, I think you might helpfully reorder the props here so that `href` is next to `to`.

Reply:
We have `href` because that's what it was and I wanted to avoid changing existing behaviour. We have `to` because that's needed for intra-application routes. I did helpfully group the ones that are conditional based on createRecordPath so it's certainly in the vicinity but I agree that it would be better to swap the order of those two. If I happen to wind up in that area again I might but otherwise I think it's not worth it.

Reply:
Agreed: best not change `href` if it was already there, on the basis of Chesterton's Fence.

I say go ahead and merge.